app/views/instrument_relays/index.html.haml

```diff
@@ -0,0 +1,41 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: @product.class.name.pluralize.underscore
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "relays"
+
+%h2= @product
+
+- if @product.relay.nil?
+  %label= t 'instruments.manage.label.control_mech'
```

Comment:
Can you update this file to use parens and double quotes?
```suggestion
  %label= t("instruments.manage.label.control_mech")
```
We don't have linters running in CI but I try to clean things up as we go.